doc/nrf/protocols.rst

```diff
@@ -12,6 +12,7 @@ They introduce you to concepts that are important to work with the protocol and
 
    ug_ble_controller
    ug_bt_mesh
+   ug_bt_qualification
```

Comment:
If we keep the proposed structure, we'll be able to keep sub-sections for mesh and qualification articles in the right-side menu. On the other hand, having qualification article right under Protocols might not be obvious for end users, hence I can create an additional directory called 'Bluetooth'.